core/src/replay_stage.rs

```diff
@@ -1812,6 +1751,14 @@ impl ReplayStage {
         }
     }
 
+    pub fn get_unlock_heaviest_subtree_fork_choice(operating_mode: OperatingMode) -> Slot {
```

Comment:
@mvines, fyi, one more of our favorite rolling upgrades 😄 . This one should be done before any of the switching proof features are enabled.

Reply:
sg, just tag me again when this lands so I can manage the rollout as clusters upgrade to 1.3